static/bidder-info/madsense.yaml

```diff
@@ -0,0 +1,12 @@
+endpoint: "https://ads.madsense.io/pbs"
```

Comment:
Endpoint is reachable 
 curl -i --location --request POST https://ads.madsense.io/pbs
HTTP/2 200 
content-length: 0